data/iatemplates/node_low_free_memory.yml

```
@@ -17,6 +17,6 @@ templates:
     for: 5m
     severity: warning
     annotations:
-      summary: Node out of memory ({{ $labels.node_name }})
+      summary: Lack of available memory on a Node ({{ $labels.node_name }})
```

I suppose it should be `the`
```suggestion
      summary: Lack of available memory on the Node ({{ $labels.node_name }})
```

---

@oter  ^

---

done